packages/xo-web/src/xo-app/hub/recipes/recipe-form-ev.js

```diff
@@ -290,6 +303,20 @@ export default decorate([
           value={value.userCompany}
         />
       </FormGrid.Row>
+      <FormGrid.Row>
+        <label>
+          <input
+            className='mt-1'
+            name='gdpr'
```

Comment:
I think the `gdpr` value should be sent to the server and the server should throw an error if `gdpr === false` (in order to handle `gdpr` for CLI use cases)